Newsletter2Go/Export/Block/Head.php

```diff
@@ -0,0 +1,174 @@
+use Magento\ConfigurableProduct\Model\ResourceModel\Product\Type\Configurable;
+
+class Head extends Template
+{
+    const NEWSLETTER2GO_SCRIPT_URL = '//static-sandbox.newsletter2go.com/utils.js'; // TODO change to production url?
+
+    /** @var  ObjectManager */
+    private $objectManager;
+
+    /**
+     * @var ProductRepositoryInterface
+     */
+    protected $productRepository;
+
+    /**
+     * @var CategoryRepositoryInterface
+     */
+    protected $categoryRepository;
+
+    public function __construct(
+        Template\Context $context,
+        array $data,
+        CategoryRepositoryInterface $categoryRepository,
+        ProductRepositoryInterface $productRepository
+    ) {
+        parent::__construct($context, $data);
+        $this->objectManager = ObjectManager::getInstance();
+        $this->categoryRepository = $categoryRepository;
+        $this->productRepository = $productRepository;
+    }
+
+    public function renderScripts()
+    {
+        $script = '';
+        $companyId = $this->_scopeConfig->getValue('newsletter_go/authentication/company_id');
+        $tracking = $this->_scopeConfig->getValue('newsletter_go/tracking/tracking_select');
+        $orderId = $this->getData('orderId');
+
+        if ($orderId && $companyId && $tracking) {
+            $order = '';
+            $orderData = $this->getOrderData($orderId);
+            $order .= 'n2g("ecommerce:addTransaction", ' . json_encode($orderData[0]) . ');';
+            $itemsData = $this->getItemData($orderId);
+            $items = $this->loopThroughItems($itemsData);
+
+            $script = '<script id="n2g_script">
+            !function(e,t,n,c,r,a,i){e.Newsletter2GoTrackingObject=r,e[r]=e[r]||function(){(e[r].q=e[r].q||[]).
+            push(arguments)},e[r].l=1*new Date,a=t.createElement(n),i=t.getElementsByTagName(n)[0],a.async=1,a.src=c,i.
+            parentNode.insertBefore(a,i)}(window,document,"script"," ' . static::NEWSLETTER2GO_SCRIPT_URL. '","n2g");
+            n2g("create", "' . $companyId . '"); ' . $order . $items . 'n2g("ecommerce:send");</script>';
+        }
+
+        return $script;
+    }
+
+    /**
+     * Returns string with json encoded items data
+     *
+     * @param array $itemsData
+     * @return string
+     */
+    private function loopThroughItems($itemsData)
+    {
+        $result = '';
+        foreach ($itemsData as $item) {
+            $item = json_encode($item);
+            $result .= 'n2g("ecommerce:addItem", ' . $item . ');';
+        }
+
+        return $result;
+    }
+
+    /**
+     * Returns array of order data that based on given order id
+     *
+     * @param int $orderId
+     * @return array
+     */
+    private function getOrderData($orderId)
+    {
+        /** @var SalesModel\Order $order */
+        $order = $this->objectManager->get(SalesModel\Order::class)->load($orderId);
+        $storeName = explode(PHP_EOL, $order->getStoreName());
+        $result[] = [
```

Comment:
Hmm... maybe simply `return [[/*that stuff*/]];`?

Also later on we do `getOrderData()[0]`, ignoring any other index... so maybe simply return a 1d array in the first place?